backend/infrastructures/config/settings/django_settings.py

```diff
@@ -145,3 +146,6 @@
         "rest_framework.permissions.AllowAny"
     ],  # Disable authentication for swagger.
 }
+
+MEDIA_URL = "/media/"
+MEDIA_ROOT = MEDIA_URL
```

Comment:
Please don't do like that.
See documentation: 
https://docs.djangoproject.com/en/5.1/ref/settings/

URL if for serving media files via API.
ROOT is for storing files.

It's not the same!

I can agree, that in this case name will be the same, but it should never be equal.
Moreover, now everything will be saved in config, which is invalid.
We should save in root directory user media.